db/wide/wide_columns_helper.cc

```diff
@@ -0,0 +1,33 @@
+//  Copyright (c) Meta Platforms, Inc. and affiliates.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "db/wide/wide_columns_helper.h"
+
+#include "db/wide/wide_column_serialization.h"
+
+namespace ROCKSDB_NAMESPACE {
+void WideColumnsHelper::DumpWideColumns(const WideColumns& columns,
+                                        std::ostream& oss, bool hex) {
+  if (hex) {
+    oss << std::hex;
+  }
+  auto it = columns.begin();
+  oss << *it;
+  for (++it; it != columns.end(); ++it) {
+    oss << ' ' << *it;
+  }
```

Comment:
This pattern relies on `columns` being non-empty, since `*columns.begin()` is not safe to do otherwise. We could either do a check here in this method, or put an assertion here and make sure it's checked at all call sites. (Doing it once here would be safer I think.)

Reply:
I'd do it here. Thanks for catching this!